dropshot/Cargo.toml

```diff
@@ -33,6 +33,7 @@ slog-json = "2.6.1"
 slog-term = "2.9.0"
 tokio-rustls = "0.23.4"
 toml = "0.5.9"
+tungstenite = "0.17.3"
```

Comment:
A lil unfortunate we have to pull tungstenite + all its deps just for one function (`derive_accept_key`). It's small enough and not really subject to change that I almost want to say we should just pull it in rather than the whole crate.